.github/workflows/ruby.yml

```diff
@@ -18,8 +18,9 @@ jobs:
           - '3.1'
           - '3.2'
           - '3.3'
+          - '3.4'
           - ruby-head
-          - jruby-9.3.3.0
+          - jruby-9.4.10.0
```

Comment:
No such version

Reply:
It should be fixed now, 9.4.10.0 it has been merged and released a couple of days ago: https://github.com/ruby/setup-ruby/pull/699